resources/common/plugins/template/cvvc-oto-gen/plugin.json

```diff
@@ -144,75 +144,32 @@
       },
       {
         "type": "float",
-        "name": "preuCV",
+        "name": "fixBuffer",
         "label": {
-          "en": "CV Preutterance (ms)",
-          "zh": "CV 的先行发声（毫秒）",
-          "ja": "CV の先行発声（ミリ秒）"
+          "en": "Initial vowel stabilization (ms)"
```

Comment:
Please add translations for zh and ja using translators. If you find it difficult I can do it later after merging.

Reply:
I can try adding the Japanese ones.